modules/learn/pages/clusters-and-availability/xdcr-enable-crossclusterversioning.adoc

```diff
@@ -0,0 +1,65 @@
+= XDCR enableCrossClusterVersioning
+:description: pass:q[Enabling the bucket property `enableCrossClusterVersioning` allows XDCR to add metadata to each replicated document.]
+
+[abstract]
+{description}
+
+
+Enabling the bucket property `enableCrossClusterVersioning` for all buckets in the replication topology is a pre-requisite for some XDCR features. The bucket property `enableCrossClusterVersioning` cannot be disabled once it is set to `true`. Therefore, you must understand all the pre-requisites and the full effects of setting `enableCrossClusterVersioning` to `true` before enabling it.
```

Comment:
Looking at this page, this paragraph seemed to have too much repetition.  Maybe reword it a bit.  Something like:

Enabling Cross Cluster Versioning for all buckets in the replication topology is a pre-requisite for some XDCR features.  The bucket property `enableCrossClusterVersioning` cannot be disabled once it is set to `true`. Therefore, you should not enable frivolously.